### ISSUE TITLE
Sorting ts import mangles file

### ISSUE THREAD

**Issue by jpaas** [snowcoders/sortier#1491]
Looks like there is still an issue with import sorting in 1.0.0...

# Language

Typescript

# Sample input source code

```
import { ExecutorContext, Executor } from "@nrwl/devkit"

interface TscExecutorOptions {
  isApp?: boolean
}

const tscExecutor: Executor<TscExecutorOptions> = async ({ isApp }, context: ExecutorContext) => {
  console.log(`isApp? ${isApp}`, context)
  return { success: true }
}

export default tscExecutor
```

# Expected output

```
import { Executor, ExecutorContext } from "@nrwl/devkit"

interface TscExecutorOptions {
  isApp?: boolean
}

const tscExecutor: Executor<TscExecutorOptions> = async ({ isApp }, context: ExecutorContext) => {
  console.log(`isApp? ${isApp}`, context)
  return { success: true }
}

export default tscExecutor
```

# Actual output (or error message)

```
import { import { ExecutorContextnrwl/devkit"

interface TscExecutorOptions {
  isApp?: boolean
}

const tscExecutor: Executor<TscExecutorOptions> = async ({ isApp }, context: ExecutorContext) => {
  console.log(`isApp? ${isApp}`, context)
  return { success: true }
}

export default tscExecutorimport { ExecutorContext, Executor, Executor } from "@nrwl/devkit"

interface TscExecutorOptions {
  isApp?: boolean
}

const tscExecutor: Executor<TscExecutorOptions> = async ({ isApp }, context: ExecutorContext) => {
  console.log(`isApp? ${isApp}`, context)
  return { success: true }
}

export default tscExecutor

```

**Comment by k2snowman69** [comment]
On it, hopefully can get something out by the EOD

**Comment by k2snowman69** [comment]
Likely dupe of 1488

**Comment by k2snowman69** [comment]
@jpaas was this via the sortier vscode plugin or via the cli? I'm having a hard time reproducing it on my cli on my windows machine... are you on a mac?

**Comment by jpaas** [comment]
I discovered it via vscode plugin, but I was also able to reproduce via CLI. I am on a mac.

**Comment by k2snowman69** [comment]
Fixed with 1.0.1

**Comment by jpaas** [comment]
Thanks @k2snowman69 . I can confirm that 1.0.1 works. It unfortunately seems to have broken the vscode extension for me, but I'll log that as a separate issue.
